connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -337,18 +337,28 @@ public void createTopic(String topic) {
      * @param topic The name of the topic.
      */
     public void createTopic(String topic, int partitions) {
-        createTopic(topic, partitions, 1, new HashMap<>());
+        createTopic(topic, partitions, 1, new HashMap<>(), new Properties());
```

Comment:
How about replace this by `createTopic(topic, partitions, 1, Collections.emptyMap());`?

Reply:
Good to me. I'll work on it tomorrow! Thanks.

Reply:
Updated. Thanks.